Dockerfile

```diff
@@ -12,7 +12,28 @@ RUN curl -SLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-
   && tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 \
   && rm "node-v$NODE_VERSION-linux-x64.tar.gz"
 
-RUN apt-get update && apt-get install -y --no-install-recommends openjdk-7-jdk
+RUN apt-get update \
+  && apt-get install -y \
+  openjdk-7-jdk \
+  apt-transport-https \
+  ca-certificates \
+  curl \
+  gnupg \
+  --no-install-recommends \
+  && curl -sSL https://dl.google.com/linux/linux_signing_key.pub | apt-key add - \
+  && echo "deb https://dl.google.com/linux/chrome/deb/ stable main" > /etc/apt/sources.list.d/google-chrome.list \
+  && apt-get update && apt-get install -y \
+  google-chrome-beta \
+  fontconfig \
+  fonts-ipafont-gothic \
+  fonts-wqy-zenhei \
+  fonts-thai-tlwg \
+  fonts-kacst \
+  fonts-noto \
+  fonts-freefont-ttf \
+  --no-install-recommends \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
Do you know how much all this new stuff adds to the size of the image? This image needs to be downloaded to every Travis node for every test run.

Reply:
According to dockerhub the previous image size was ~570MB.

Reply:
locally it's ~1.72GB...going to see what the compressed version on dockerhub looks like.